### PR TITLE
Adding Swag Submission Page

### PR DESCRIPTION
**Pull request by LiamRandall**

### CODE REVIEW

File: swag/swag.mdx
Comment (brooksmtownsend) [inline swag/swag.mdx:1]: I think this can be removed

File: docusaurus.config.ts [hunk 1]
```diff
@@ -176,6 +176,7 @@ const config = (async (): Promise<Config> => {
           { to: '/blog', label: 'Blog', position: 'left' },
           { to: '/community', label: 'Community', position: 'left' },
           { type: 'doc', docId: 'intro', position: 'left', label: 'Docs' },
+/*          { type: 'doc', docId: 'swag', position: 'left', label: 'Get Swag' },*/
```
Comment (brooksmtownsend) [inline docusaurus.config.ts]: This can be removed

File: docs/swag.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+title: 'wasmCloud Swag'
+sidebar_position: 20
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import HubspotForm from 'react-hubspot-form';
+
+
+# wasmCloud Swag
```
Comment (brooksmtownsend) [inline docs/swag.mdx]: I think we should write out a bit more on this page, what it's for, information folks need to provide, etc

Comment (brooksmtownsend) [reply docs/swag.mdx]: Should this be open only to folks that have at least one good-first-issue closed? US Residents only?

**Review by brooksmtownsend — DISMISSED** [review]
Just a request for more info

File: docusaurus.config.ts (hunk 1 above)
Comment (lachieh) [reply docusaurus.config.ts]: Removed comment and added link to footer instead

File: docs/swag.mdx (hunk 2 above)
Comment (lachieh) [reply docs/swag.mdx]: There is some extra information on the HubSpot form itself. It's only open to US folks, but we could include a comment to say not all submissions are guaranteed a swag-kit and contributors are prioritized so be sure to include your github id?

**Comment by lachieh** [comment]
@brooksmtownsend @LiamRandall I've adjusted some of the language on the form, added a t-shirt size and moved the page to `/swag`. I also added a link in the footer instead of in the docs sidebar.

https://deploy-preview-550--dreamy-golick-5f201e.netlify.app/swag

File: swag/swag.mdx
Comment (brooksmtownsend) [inline swag/swag.mdx:1]: I think we can remove this right?

**Review by brooksmtownsend — COMMENTED** [review]
Looks good to me, just looks like theres still an extra page

File: swag/swag.mdx
Comment (lachieh) [reply swag/swag.mdx:1]: yes, missed that, removed

**Review by lachieh — APPROVED** [review]

**Review by lachieh — APPROVED** [review]

**Pull request merged**